remoteappmanager/static/js/home/configurables.js

```diff
@@ -1,85 +1,59 @@
 define([
-    "jquery",
-    "handlebars",
-    "utils"
-], function($, hb, utils) {
+    "utils",
+    "components/vue/dist/vue.min"
+], function(utils, Vue) {
     "use strict";
 
-    var view_template = hb.compile(
-        '<div class="form-group">' +
-        '<label for="resolution-model-{{index}}">Resolution</label>' +
-        '<select class="form-control" id="resolution-model-{{ index }}">' +
-        '{{#each options}}' +
-           '<option value="{{this}}">{{this}}</option>' +
-        '{{/each}}' +
-        '</div>'
-    );
+    var resolution_conf_tag = 'resolution';
+    var resolution_component = Vue.component(resolution_conf_tag + '-component', {
+        // Your configurable must have a "value" property
+        props: ['value'],
 
-    var ResolutionModel = function () {
-        // Model for the resolution configurable.
-        var self = this;
-        this.resolution = "Window";
-        this.resolution_options = ["Window", "1920x1080", "1280x1024", "1280x800", "1024x768"];
-       
-        self.view = function (index) {
-            // Creates the View to add to the application entry.
-            var widget = $(view_template({
-                options: self.resolution_options,
-                index: index
-            }));
-            
-            widget.find("select").change(function() {
-                if (this.selectedIndex) {
-                    self.resolution = this.options[this.selectedIndex].value;
+        template:
+            '<div class="form-group">' +
+            '  <label>Resolution</label>' +
+            '  <select class="form-control" v-model="selected_value">' +
+            '    <option v-for="resolution_option in resolution_options">' +
+            '      {{resolution_option}}' +
+            '    </option>' +
+            '  </select>' +
+            '</div>',
+
+        data: function() {
+            return {
+                selected_value: this.value,
+                resolution_options: ['Window', '1920x1080', '1280x1024', '1280x800', '1024x768']
+            };
+        },
+
+        watch: {
+            value: function() {this.selected_value = this.value;}, // model -> view update
+            selected_value: function() {this.$emit('update:value', this.selected_value);} // view -> model update
+        },
+
+        methods: {
+            as_config_dict: function() {
```

Comment:
this should be in the model. All your interaction should be with the model, and the component is just a view that is inserted in the right place.